kratos/utilities/brute_force_point_locator.h

```diff
@@ -23,6 +23,25 @@
 
 namespace Kratos
 {
+
+///@name Enums
+///@{
+
+/**
+ * @brief Enum for Initial and Current configurations
+ * @note This enum is common enough to justify being global, and should be
+ * moved to a dedicated enum header
+ */
+enum struct Configuration
+{
+    Initial = 0,
+    Current = 1
+};
```

Comment:
can you move this to here please?
https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/includes/global_variables.h